Assets/Plugins/CountlySDK/Services/DeviceIdCountlyService.cs

```diff
@@ -28,7 +28,7 @@ internal void InitDeviceId(string deviceId = null)
         {
             //**Priority is**
             //Cached DeviceID (remains even after after app kill)
-            //Static DeviceID (only when the app is running either backgroun/foreground)
+            //Static DeviceID (only when the app is running in either background/foreground)
```

Comment:
running or background

Reply:
Which one is better 1 or 2?

only when the app is running or in the background

only when the app is running or background

Reply:
#1